troi/__init__.py

```diff
@@ -219,6 +219,23 @@ def __str__(self):
         return "<Recording('%s', %s, %s)>" % (self.name, self.mbid, self.msid)
 
 
+class Playlist(Entity):
+    """
+        The class that represents a playlist, which is nothing more than a [Recording] with metadata.
+    """
+    def __init__(self, name=None, mbid=None, filename=None, recordings=[],
```

Comment:
bad idea to set a default parameter as `[]`, use None instead and in the code do `self.recordings = recordings or []`

https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments